src/containers/ProvisionerForm.js

```diff
@@ -8,6 +8,8 @@ import GithubService from '../components/provisionerForm/GithubService';
 import ProjectName from '../components/provisionerForm/ProjectName';
```

Comment:
This import just works for dev, for prod a minified file is generate with *all* the imports.